app/models/host.rb

```diff
@@ -1881,5 +1881,16 @@ def read_only_storages
     storages.where(:host_storages => {:read_only => true})
   end
 
+  def archived?
+    ems_id.nil?
+  end
+  alias_method :archived, :archived?
```

Comment:
Ah, ok. But  this looks like it's needed for the virtual attribute. So I'd rather not include it here, but just my 2 :euro: